components/Faucet.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, useState } from "react";
+import { css } from "@emotion/react";
+import AccountIdenticon from "@/components/AccountIdenticon";
+import CompareArrowsIcon from "@mui/icons-material/CompareArrows";
+import { useSession } from "next-auth/react";
+import FaucetButton from "@/components/FaucetButton";
+
+const Faucet: FC = () => {
+	const { data: session } = useSession();
+	const [localNetwork, setLocalNetwork] = useState<Boolean>(false);
+	const [address, setAddress] = useState<string>("");
+
+	return (
+		<div css={styles.faucetWrapper}>
+			<div css={styles.faucetContainer}>
+				<p css={styles.heading}>
+					Enter your CENNZnet Address to receive testnet tokens:
+				</p>
+				<div css={styles.addressWrapper}>
+					<AccountIdenticon
+						css={styles.accountIdenticon}
+						theme="beachball"
+						size={28}
+						value={
+							address
+								? address
+								: "5FbMzsoEpd2mt8eyKpKUxwJ5S9W7nJVJkCer2Jk7tvSpB1vF"
+						}
+					/>
+					<input
+						css={styles.input}
+						type="text"
+						placeholder="5FbMzsoEpd2mt8eyKpKUxwJ5S9W7nJVJkCer2Jk7tvSpB1vF"
+						value={address}
+						onChange={(e) => setAddress(e.target.value)}
+						disabled={!!session?.validAccount}
+					/>
+				</div>
+				<div css={styles.networks}>
```

Comment:
looks good man although thinking maybe we have a dropdown box like cennznet.io? Also think we should have rata/local/nikau all separate requests.